frontend/src/components/Bill/Bill.css

```diff
@@ -0,0 +1,73 @@
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+3:ital,wght@0,200..900;1,200..900&display=swap');
+
+@media screen and (max-width: 1000px) {
+    .bill-content {
+        display: flex;
+        flex-direction: column;
+    }
+    .summary {
+        display: flex;
+        flex-direction: column;
+        overflow: auto;
+        background: #eeede7;
+        max-height: max-content;
+        min-height: min-content;
+        border-radius: 10px;
+        margin: 16px;
+        padding: 20px;
+    }
+}
+
+html {
```

Comment:
for later: we should set up fonts in the main css instead of here + avoid manipulating generic html tags (ex here: html) in these component-related css files

Reply:
Agreed! Sorry I missed these comments